build.gradle

```diff
@@ -114,10 +114,15 @@ dependencies {
     implementation 'net.logstash.logback:logstash-logback-encoder:6.6'
     implementation 'net.javacrumbs.shedlock:shedlock-spring:4.25.0'
     implementation 'net.javacrumbs.shedlock:shedlock-provider-jdbc-template:4.25.0'
+    implementation 'com.sun.istack:istack-commons-runtime:3.0.12'
```

Comment:
This was an indirect dependency we got via `com.sun.xml.bind:jaxb-xjc`. We now need to declare it as a direct dependency of our project.